package.json

```diff
@@ -1,37 +1,65 @@
 {
-    "name": "data-visualizer",
-    "description": "DHIS2 Data Visualizer",
-    "private": true,
+    "name": "data-visualizer-app",
     "version": "39.2.13",
-    "workspaces": [
-        "packages/*"
-    ],
+    "description": "DHIS2 Data Visualizer",
     "license": "BSD-3-Clause",
+    "private": true,
+    "scripts": {
+        "build": "d2-app-scripts build",
+        "start": "d2-app-scripts start",
+        "test": "d2-app-scripts test",
+        "coverage": "yarn test --coverage",
+        "deploy": "d2-app-scripts deploy",
+        "lint": "d2-style check",
+        "format": "d2-style apply",
+        "validate-commit": "d2-style check --staged",
+        "validate-push": "yarn test",
+        "cypress:capture": "start-sever-and-test 'yarn start' http://localhost:3000 'yarn cypress run --env networkMode=capture'",
```

Comment:
I wonder if `cypress:capture` and `cypress:stub` even work right now? If not, then they can be removed until we start doing testing with fixtures.

Reply:
Could you also add a cypress:run command so the full test suite can be run without opening the Cypress gui?  e.g., ```"cypress:run": "start-server-and-test 'yarn start' http://localhost:3000 'yarn cypress run --env networkMode=live'"```